subprojects/internal-testing/src/main/groovy/org/gradle/test/fixtures/file/TestFile.java

```diff
@@ -641,6 +641,10 @@ public boolean isSelfOrDescendent(File file) {
         if (file.getAbsolutePath().equals(getAbsolutePath())) {
             return true;
         }
+        return isDescendent(file);
+    }
+
+    public boolean isDescendent(File file) {
```

Comment:
⛏️ `isDescendant` (that's the usual spelling for this relationship in the code base).

Reply:
I agree, but then would become inconsistent with the method it was extracted from. I will rename the other one as well, only one call site anyways